DEVELOPMENT.md

```diff
@@ -0,0 +1,93 @@
+## Issues
```

Comment:
I feel like this file has good information in it but doesn't belong in platform-examples directly. I suspect you checked this in by accident?

These look like they are all valid bugs if I'm reading this correctly?
Can you remove this file from the branch, turn these into shortcuts, and add them to the product feedback meeting to highlight as rough edges on the replicated CLI that we can highlight to ... the vendor portal team I think.